### DISCUSSION TITLE
Mass delete 0;0 points?

### DISCUSSION THREAD

**Discussion by Kiriyashiki** [Freika/dawarich#459] [Q&A] [answered]
Ive been using the new Trips feature, and came across an issue like this :
![image](https://github.com/user-attachments/assets/eeda66b0-3adb-4da9-96da-e93fb7d19fb5)

My stats indicate Ive travalled 1 million kilometers all time, which is way too much. I think I must have a lot of 0;0 points in my data. Is there any efficient way of deleting all 0;0 points from all my data?

**Comment by Freika** [comment, marked as answer]
In the console (https://dawarich.app/docs/FAQ#how-to-enter-dawarich-console)

Run this:

```ruby
points = Point.where('latitude = 0.0 OR longitude = 0.0')
points.size #=> will return number of such points
points.destroy_all #=> will delete all these points
```

**Comment by Kiriyashiki** [comment]
Thanks for the help!
